### PR TITLE
Refactor calculator to decouple the UI from the busniess logic.

### PR DESCRIPTION
**Pull request by javicavi**

### CODE REVIEW

**Comment by jose-carmona-villegas** [comment]
Use [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) (VSCode plugin) to setup some code analysis rules:
https://www.reddit.com/r/vscode/comments/e8e1hs/show_unused_js_code/

**Comment by jose-carmona-villegas** [comment]
Please, clean up your unused commented code.

**Comment by javicavi** [comment]
> Please, clean up your unused commented code.

**Comment by jose-carmona-villegas** [comment]
script.js is already a very large file. Maybe you could split it in two, one for visuals and another for business logic.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    {
+      "type": "chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:5500",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```
Comment (jose-carmona-villegas) [inline .vscode/launch.json]: Since this file was added to the gitignore (as it's a local config file), we should remove it from the PR.

**Review by jose-carmona-villegas — COMMENTED** [review]
We'll segregate the UI and the business logic further in another PR.

**Review by jose-carmona-villegas — APPROVED** [review]

**Pull request merged**